stories/IntroStory.tsx

```diff
@@ -1,4 +1,5 @@
 import * as React from 'react';
+const { version } = require('../package.json');
```

Comment:
not sure why `process.env.npm_package_version` is undefined...

Reply:
Oh cool I didn't know you can do this haha